oc-adm-util/.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```

Comment:
syntax error!

Reply:
you mean the comments? That's fine, it's generated by vscode and only used by it.